src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -285,7 +289,9 @@
     paddingLeft: normalize(20),
     justifyContent: 'space-between',
     alignItems: 'center',
-    backgroundColor: COLORS.almostBlack
+    backgroundColor: COLORS.almostBlack,
+    borderBottomColor: COLORS.gray310,
+    borderBottomWidth: 1
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the element has been changed from `COLORS.almostBlack` to `COLORS.gray110`, and a border bottom has been added with color `COLORS.gray310` and width 1. These changes can enhance the visual distinction of the element.
```diff
+     backgroundColor: COLORS.almostBlack,
+     borderBottomColor: COLORS.gray310,
+     borderBottomWidth: 1
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->